addons/calendar/models/calendar_event.py

```diff
@@ -206,6 +205,7 @@ def _compute_is_highlighted(self):
     show_as = fields.Selection([('free', 'Free'), ('busy', 'Busy')], 'Show Time as', default='busy', required=True)
 
     # linked document
+    # LUL TODO use fields.Reference ?
```

Comment:
Well it depends. Is an end user supposed to configure those field manually on the form view ? 

Or do we expect them to always use the calendar view + the action buttons,... that are offered to him?

If the second sentence is correct, I wouldn't use reference fields here. What do you think ?